test/fixtures/index_choices.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  index_id: 1
+  choice_id: 1
```

Comment:
these fixtures are invalid, they should be using 
```
one:
  index: one
  choice: one
```

and then the second one should probably reference choise two